tools/cdf-lineage-exporter/requirements.txt

```diff
@@ -0,0 +1,6 @@
+flask
+google-cloud-storage
+black
```

Comment:
curious why preference for black over yapf?
The style check for this repo checks against yapf.

Also we should probably remove black, isort, and pytest from this because there's no need for the runtime (cloud function) to use these static code / dev tools.

Thoughts on moving these to a `dev-requirements.txt` in the root dir of this example?

Reply:
`black` is what Cloud DevRel has standardized on for all libraries and samples. Okay to switch back if professional services prefers `yapf`.

`dev-requirements.txt` is a good idea.

Reply:
Yeah the (currently failing) CI check uses `yapf --style=google`.
https://github.com/GoogleCloudPlatform/professional-services/blob/master/helpers/check_format.sh#L39